cypress/e2e/api/testsApi.cy.js

```diff
@@ -0,0 +1,162 @@
+describe('Validate the API requests', () => {
+  const CUSTOMERS_API_URL = `${Cypress.env('API_URL')}/customers`
+  context('Postive scenarios', () => {
+    context('Pagination and limits', () => {
+      it('Get customers successfully', () => {
+        cy.request({
+          method: 'GET',
+          url: `${CUSTOMERS_API_URL}?page=1&limit=10`,
```

Comment:
Aqui você não precisaria passar `page=1&limit=10` visto que estes já são os valores padrão para este _endpoint_.

Podia ser só:

```suggestion
          url: CUSTOMERS_API_URL,
```

E iria funcionar igual.

Menso código e mais simples!